### PR TITLE
Fix autowiring for upcoming Symfony 3.3

### PR DESCRIPTION
**Pull request by chalasr**
pending https://github.com/symfony/symfony/pull/22223

### CODE REVIEW

**Comment by chalasr** [comment]
https://github.com/symfony/symfony/pull/22223 merged, green again.

File: Resources/doc/3-functional-testing.md [hunk 1]
```diff
@@ -7,17 +7,17 @@ Configuration
 Generate some test specific keys, for example :
 
 ``` bash
-$ openssl genrsa -out app/var/jwt/private-test.pem -aes256 4096
-$ openssl rsa -pubout -in app/var/jwt/private-test.pem -out app/var/jwt/public-test.pem
+$ openssl genrsa -out var/jwt/private-test.pem -aes256 4096
+$ openssl rsa -pubout -in var/jwt/private-test.pem -out var/var/jwt/public-test.pem
```
Comment (lemoinem) [inline Resources/doc/3-functional-testing.md]: I think there is one "var" too many in `-out` ;)

**Comment by lsmith77** [comment]
would be nice to get this merged soonish .. as I am already using `json_login` from sf 3.3 and it seems like lots of work is happening in master of this bundle that isn't showing up in this branch.

I guess atm there are issues with sf 2.8 that are left to fix?

```
The Symfony\Component\Security\Core\Util\SecureRandom class is deprecated since version 2.8 and will be removed in 3.0. Use the random_bytes() function instead: 1x
    1x in AutowiringTest::testAutowiring from Lexik\Bundle\JWTAuthenticationBundle\Tests\Functional\DependencyInjection
```

**Comment by chalasr** [comment]
@lsmith77 the remaining failure has been fixed in https://github.com/symfony/symfony/pull/22282 but not yet released. 
I relaxed travis until the next symfony maintenance releases, merging this one

**Comment by lsmith77** [comment]
thx

**Pull request merged**